lib/circuitry/provisioning/provisioner.rb

```diff
@@ -14,20 +13,21 @@ def run
         queue = create_queue
         return unless queue
 
-        create_topics(:publisher, config.publisher_topic_names)
-        subscribe_topics(queue, create_topics(:subscriber, config.subscriber_topic_names))
+        create_topics(:publisher, Circuitry.publisher_config.topic_names)
+        subscribe_topics(queue, create_topics(:subscriber, Circuitry.subscriber_config.topic_names))
```

Comment:
:neckbeard: I feel like a lot of lines in this class would read better if we created helper methods:

```ruby
def publisher_config
  Circuitry.publisher_config
end

def subscriber_config
  Circuitry.subscriber_config
end
```